apps/i18n/spritelab/en_us.json

```diff
@@ -137,14 +147,21 @@
+  "removeOnlyTouchedSprite": "All of the sprites were removed instead of just the one that was touch. Try using the `subject sprite` or `object sprite` blocks block with the `remove` block.",
   "sayBlock": "Type something for the sprite to say in a `say` block. \n<xml><block type=\"gamelab_spriteSay\"><title name=\"SPEECH\">Hello, world!</title><value name=\"SPRITE\"><block type=\"gamelab_allSpritesWithAnimation\"><title name=\"ANIMATION\">\"default\"</title></block></value></block></xml>",
+  "saySprite": "The sprite didn't say anything.",
+  "sayVariable": "Make the sprite say the value of your variable. Be sure the label matches your code. \n <xml><block type=\"gamelab_spriteSayTime\">\n    <value name=\"SPRITE\">\n      <block type=\"gamelab_allSpritesWithAnimation\">\n        <field name=\"ANIMATION\">\"purple bunny\"</field>\n      </block>\n    </value>\n    <value name=\"TEXT1\">\n      <block type=\"variables_get\">\n        <field name=\"VAR\">count</field>\n      </block>\n    </value>\n    <value name=\"NUM\">\n      <block type=\"math_number\">\n        <field name=\"NUM\">4</field>\n      </block>\n    </value>\n  </block></xml>",
   "secondClickButNoPrint": "You clicked on more than one sprite but no text was printed. Add a `print` block under each `when clicked` event block.",
   "secondClickButNoSpeech": "You clicked on more than one sprite but one of your clicks didn't cause a sprite to say anything. Add a `say` block under each `when clicked` event. \n<xml><block type=\"gamelab_spriteSay\"><title name=\"SPEECH\">Hello, world!</title><value name=\"SPRITE\"><block type=\"gamelab_allSpritesWithAnimation\"><title name=\"ANIMATION\">\"default\"</title></block></value></block></xml>",
   "setSecondSpriteSize": "Use a `set size` block to change the size of the sprite. Use a size other than 100. \n <xml><block type=\"gamelab_setProp\"><title name=\"PROPERTY\">\"scale\"</title><value name=\"SPRITE\"><block type=\"gamelab_allSpritesWithAnimation\"><title name=\"ANIMATION\">\"flower2_1\"</title></block></value><value name=\"VAL\"><block type=\"math_number\"><title name=\"NUM\">50</title></block></value></block></xml>",
   "shareGame": "Share your game:",
   "shipBadPosition": "Use the location block to move the ship closer to the top of the screen. <xml><block type=\"gamelab_location_picker\"><title name=\"LOCATION\">'{\"x\":100,\"y\":275}'</title></block></xml>",
   "shipStayedInBounds": "Your sprite should move off of the screen. Try using a behavior like \"moving east\".",
   "singleSpriteNotClicked": "Make sure that you click on a sprite and that you have the right events in your workspace. \n<xml><block type=\"gamelab_spriteClicked\"><title name=\"CONDITION\">\"when\"</title></block></xml>",
+  "sayUseClicked": "All of your sprites spoke instead of just the one that was clicked. Try using the `clicked sprite` block inside your event block.",
```

Comment:
Someones we talk about "your sprite(s)" and sometimes "the sprite(s").  Is there a rule?